test/data/scroller.js

```diff
@@ -0,0 +1,20 @@
+var items = [
+  {
+    "text": "Did Pluto have rivers and lakes?<br clear='all'/><br/><br/>",
```

Comment:
@donnapep Are the `<br/>` tags necessary for every `item.text` we pass? If so, can they be appended instead of part of the text we pass in?